android/pytorch_android/src/main/java/com/facebook/pytorch/PytorchScriptModule.java

```diff
@@ -0,0 +1,34 @@
+package com.facebook.pytorch;
+
+import com.facebook.jni.HybridData;
+
+import java.nio.Buffer;
+
+public class PytorchScriptModule {
+
+  private NativePeer mNativePeer;
+
+  public PytorchScriptModule(final String modelAbsolutePath) {
+    this.mNativePeer = new NativePeer(modelAbsolutePath);
+  }
+
+  public Tensor run(Tensor input) {
+    return mNativePeer.run(input.getRawDataBuffer(), input.dims, input.getTypeCode());
+  }
+
+  private static class NativePeer {
+    static {
+      System.loadLibrary("pytorch_android");
```

Comment:
any reason it can't be called just `libtorch` or `pytorch`? (or whatever the naming convension is on java :) )

Reply:
We already have libtorch and libc10 libs, just not to intersect names with them and anything will be ok, it tries to load lib${name}, 
libpytorch and System.loadLibrary("pytorch") ok or better to keep this name for something else?